### ISSUE TITLE
Dark theme not supported

### ISSUE THREAD

**Issue by kinow** [PauloMigAlmeida/directed-acyclic-graph-builder-js#4]
**Describe the bug**

Switching to a dark theme with Dark Reader works, except for the icons when you drag and drop them onto the canvas area.

**To Reproduce**
1. Switch to dark more
2. Note how everything was rendered in dark mode
3. Now drag and drop a graph node

**Expected behavior**

The node was supposed to render following the browser set preference (annoying, but with some CSS I guess that's doable)

**Tooling version:**
 - Node.js version: v16.17.0 (probably doesn't matter as I used the demo)
 - NPM version: v16.17.0 (ditto :point_up: )
 - Browser: 104.0.1 (64-bit), Linux Ubuntu LTS, Dark Reader extension enabled

**Additional context**

![GIFrecord_2022-09-06_224602](https://user-images.githubusercontent.com/304786/188621651-177fed6d-bbcd-44ae-9d43-a08e63bd7786.gif)

(From https://github.com/PauloMigAlmeida/directed-acyclic-graph-builder-js/issues/2#issuecomment-1237985291)

**Comment by PauloMigAlmeida** [comment]
@kinow Is this the Extension that you are using? https://addons.mozilla.org/en-US/firefox/addon/darkreader/

**Comment by kinow** [comment]
> @kinow Is this the Extension that you are using? https://addons.mozilla.org/en-US/firefox/addon/darkreader/

Yup, that's the one, with that robot icon. I think it was in the list of recommended extensions in Firefox. I use it at night when my wife goes to sleep, as the light of the monitor is too bright (small apartment :grimacing: )

**Comment by PauloMigAlmeida** [comment]
@kinow Been there! The older I'm getting, the more sensitive to light my eyes are becoming.... I hope this isn't a sign of glaucoma gearing towards my way 😨 

Re: the dark mode issue, it looks like DarkReader has some known issues with SVG elements given the way colours are specified (which differ from regular html elements): Ex.: https://github.com/darkreader/darkreader/issues/6255

I will leave this issue opened, but I think the best way I can help here would be by opening a PR on darkreader's repo to fix the root cause of the issue instead.
